README.md

```diff
@@ -28,6 +28,8 @@ Install-Module AzureAd
 # Deployment
 
 To deploy the sample scenario, first clone this git repo and find the deployment scripts folder:
+**NOTE** You must use Windows Powershell, not Powershell Core! 
```

Comment:
This is not actually true. The issue is that if you don't provide a password (with the `-AdminPassword` parameter), the script will try to generate one. The module for doing that is not in PowerShell Core (yet). So it will work in PowerShell Core if you use `-AdminPassword $(ConvertTo-SecureString -AsPlainText -Force "MyCrazyPassword!!")`.